### ISSUE TITLE
[Website] Screen Reader conveys  incorrect name for the ‘Close’ button under video pane.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6422]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/  9/8/2021

### Application Name

Website

### Problem Description

[ADO 35879770](https://microsoft.visualstudio.com/OS/_workitems/edit/35879770)

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to "watch Video" button & activate it with ENTER key.
3. Navigate to ‘Close’ button and observe the issue.

**Actual Result:**
Screen Reader conveys incorrect name for the ‘Close’ button under video pane. It is announcing as “Times Group”.

**Expected Result:**
Screen Reader should convey the correct information for the ‘Close’ button and it should announce as ‘Close’ button.

### Card JSON

```JSON
No card
```
